src/pages/home/report/ReportActionCompose.js

```diff
@@ -62,6 +70,11 @@ class ReportActionCompose extends React.Component {
         if (this.props.comment && prevProps.comment === '' && prevProps.comment !== this.props.comment) {
             this.comment = this.props.comment;
         }
+
+        // When any modal goes from visible to hidden, bring focus to the compose field
+        if (prevProps.modal.isVisible === true && this.props.modal.isVisible === false) {
```

Comment:
NAB. We can just do `if (prevProps.modal.isVisible && !this.props.modal.isVisible)` here, right?

Reply:
Updated!